docs/keyboardShortcuts.md

```diff
@@ -73,4 +74,4 @@ These shortcuts can be used when the focus is on an issue card in a panel.
 | ------------- |:-------------:| -----------|
 | Scroll to top|<kbd>I</kbd>|Scrolls to the top of the page loaded in browser view|
 | Scroll to bottom|<kbd>N</kbd>|Scrolls to the bottom of the page loaded in browser view|
-| Scroll up/down|<kbd>J</kbd> / <kbd>K</kbd>|Scroll up/down the page loaded in browser view|
\ No newline at end of file
+| Scroll up/down|<kbd>J</kbd> / <kbd>K</kbd>|Scroll up/down the page loaded in browser view|
```

Comment:
Yup even I'm surprised why Github is displaying this as a change. There seems to be no change. It's exactly the same thing.